src/index.html

```diff
@@ -5,8 +5,402 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="header">
+      <img
+        src="images/logo.png"
+        alt="logo-icon"
+        class="header__logo"
+      >
+      <nav class="nav header__nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#apple"
+              class="nav__link is-active"
+              >apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#sansung"
+              class="nav__link"
+              >samsung
+
```

Comment:
текст потрібно переносити на нову лінію коду, так потрібно пофіксити всі лінки
```suggestion
            >
              samsung

```